entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "RECREATE NGINX CONFIG"
+envsubst < /etc/nginx/conf.d/nginx.template > /etc/nginx/conf.d/default.conf
+echo "START NGINX"
+service nginx start
+echo "START NODE"
+./build/ava --db-dir /db
```

Comment:
consider to make the configuration of the ava client more flexible by:
1. adding a new parameter in the docker-compose.yml like `AVA_CMD_OPTIONS` that will contain `--db-dir /db` only so far
2. changing this line to <path>/ava $AVA_CMD_OPTIONS

The idea is to have an ability to provide more parameters to the client without necessity to rebuild the image